src/main/java/org/foundation/coding/customer/Customer.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Refactor Code Tasks.
+ */
+package org.foundation.coding.customer;
+
+import java.util.ArrayList;
+import org.foundation.coding.movie.ChildrenMovie;
+import org.foundation.coding.movie.Movie;
+import org.foundation.coding.movie.NewReleaseMovie;
+import org.foundation.coding.movie.RegularMovie;
+import org.foundation.coding.rental.Rental;
+
+/**
+ * The class contains info for rental movie.
+ */
+public class Customer {
+  //Variable to define the children movie.
+  private static final int CHILDREN = 2;
+  //Variable to define the regular movie.
+  private static final int REGULAR = 0;
+  //The name of the customer.
+  private String name;
+
+  /**
+   * Return a type of list rental.
+   * @return rental list.
+   */
+  public ArrayList<Rental> getRentals() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)